.github/workflows/build-image.yml

```diff
@@ -0,0 +1,47 @@
+name: Create and publish a Docker image
+
+on:
+  push:
+    branches: ['imikushin/automate-image-build'] # TODO change to main
```

Comment:
change to main I guess :D

Reply:
altho should we maybe not do this on every push, but rather for releases specifically?

https://docs.github.com/en/actions/publishing-packages/publishing-docker-images has this example:

```yml
on:
  release:
    types: [published]
```

Reply:
> change to main I guess :D

Will do: I'm still experimenting 🙃

Reply:
Updated: only pushing to `release` will trigger building and publishing an image.